x-pack/plugins/enterprise_search/public/applications/shared/http/http_logic.ts

```diff
@@ -11,25 +11,26 @@ import { HttpSetup, HttpInterceptorResponseError, HttpResponse } from 'src/core/
 
 import { ERROR_CONNECTING_HEADER, READ_ONLY_MODE_HEADER } from '../../../../common/constants';
 
-interface HttpValues {
+export interface HttpValues {
```

Comment:
I needed this interface for a test at https://github.com/elastic/kibana/pull/122044/files#diff-b4f890a051f892ac373d560f32757bd075433cf5a1bdcc04ad26d1faee2949e2R16

Reply:
This is really a tiny nitpick, but given that you're only using `Partial<HttpValues>` there and not the entire object, you could just explicitly define it locally in that file and save the export (which could have some very minor implications for build size/time). But that really is nitpicking.